src/components/MaintenanceBanner/MaintenanceBanner.stories.jsx

```diff
@@ -14,34 +14,29 @@ export default {
   },
 };
 
-const Template = args => {
-  const startTime = moment.utc(moment(args.startsAt));
-  const endTime = moment.utc(moment(args.expiresAt));
-  const warnStart = moment.utc(moment(args.warnStartsAt));
+const Template = (args) => {
+  const startTime = args.startsAt;
+  const endTime = args.expiresAt;
+  const warnStart = args.warnStartsAt;
   console.group('Times passed to Template');
-  console.log('startTime:', startTime.format('MM/DD/YY HH:mm'));
-  console.log('endTime', endTime.format('MM/DD/YY HH:mm'));
-  console.log('warnStart', warnStart.format('MM/DD/YY HH:mm'));
+  console.log('startTime:', format(startTime, 'MM/dd/yy p'));
+  console.log('endTime', format(endTime, 'MM/dd/yy p'));
+  console.log('warnStart', format(warnStart, 'MM/dd/yy p'));
   console.groupEnd();
-  return (
-    <MaintenanceBanner
-      {...args}
-      startsAt={startTime}
-      expiresAt={endTime}
-      warnStartsAt={warnStart}
-    />
-  );
+  return <MaintenanceBanner {...args} />;
 };
 
 const defaultArgs = {
   id: 'maintenence-banner-id',
-  title: 'Current maintenance notice',
-  warnTitle: 'Future maintenance notice',
-  content: 'The site is currently under maintenance.',
-  warnContent: 'The site will be undergoing maintenance shortly.',
-  startsAt: moment().valueOf(),
-  expiresAt: moment().add(1, 'hour').valueOf(),
-  warnStartsAt: moment().subtract(30, 'minutes').valueOf(),
+  title: 'Site maintenance',
+  warnTitle: 'Upcoming site maintenance',
+  content:
+    'We’re working on VA.gov right now. If you have trouble signing in or using tools, check back after we’re finished. Thank you for your patience.',
+  warnContent:
+    'We’ll be doing some work on VA.gov. The maintenance will last 1 hour. During that time, you won’t be able to sign in or use tools.',
+  startsAt: new Date(Date.now()),
+  expiresAt: addHours(Date.now(), 1),
+  warnStartsAt: subMinutes(Date.now(), 30),
```

Comment:
The content changes here are for the original request in @rtwell's issue comment.